### PR TITLE
remove BPMN rest api csrf config

### PR DESCRIPTION
**Pull request by Subasinghe**

### CODE REVIEW

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 1]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
         return strBuilder.toString();
     }
 
+    public static int getTotalInstance(ProcessInfoType processInfo) {
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: need doc comments.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 2]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
...
+
+        return totalInstances;
+    }
+
+    public static String getInstanceFilterURL(String processId) {
+        StringBuilder strBuilder = new StringBuilder();
+        strBuilder.append("list_instances.jsp?filter=pid=");
+        strBuilder.append(processId);
+        strBuilder.
+                append(" status=active|completed|suspended|terminated|failed|error&order=-last-active");
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: These status are in constants AFAIR. Use them if exist or better to add them.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 3]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
...
+                    processDeployDetailsListType.getProcessEventsList().getEnableEventsList();
+            if (enableEventListType.getEnableEvent() != null) {
+                enabledEvents = enableEventListType.getEnableEvent();
+            }
+        }
+
+        return enabledEvents;
+    }
+
+    public static ScopeEventType[] setScopeEventsList(
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Add comments please.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 4]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
...
+            ProcessDeployDetailsList_type0 processDeployDetailsListType) {
+        List<String> successCategories = null;
+        if (processDeployDetailsListType.getCleanUpList() != null &&
+                processDeployDetailsListType.getCleanUpList().getCleanUp() != null) {
+            successCategories = new ArrayList<String>();
+            CleanUpType[] cleanUpTypes = processDeployDetailsListType.getCleanUpList().getCleanUp();
+
+
+            for (CleanUpType cleanUpType : cleanUpTypes) {
+                if (cleanUpType.getOn().getValue().equalsIgnoreCase("success") &&
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Is cleanUpType.getOn() nullable??

File: components/bpmn/bpmn-explorer-web/src/web/jaggery.conf [hunk 5]
```diff
@@ -94,7 +94,7 @@
              "url":"/backendRequest/*",
              "path":"/controller/backendRequest.jag"
         },
-		{
+		    {
```
Comment (hasithaa) [inline components/bpmn/bpmn-explorer-web/src/web/jaggery.conf]: Formatting issues.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 6]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
...
+
+        memInfo.put("UsedMemoryValue", usedMem.getValue());
+        memInfo.put("UsedMemoryUnit", usedMem.getUnit());
+        memInfo.put("TotalMemoryValue", totalMem.getValue());
+        memInfo.put("TotalMemoryUnit", totalMem.getUnit());
+
+        return memInfo;
+    }
+
+    public static String createLongRunningInstanceJSONString(PaginatedInstanceList longRunningInstances) {
```
Comment (hasithaa) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Add Java Doc to all public methods.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java (hunk 1 above)
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java (hunk 2 above)
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java (hunk 3 above)
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java (hunk 4 above)
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

File: components/bpmn/bpmn-explorer-web/src/web/jaggery.conf [hunk 7]
```diff
@@ -78,7 +78,7 @@
             "url":"/completedTask/*",
             "path":"/controller/completedTaskController.jag"
         },
-		{
+		    {
```
Comment (Subasinghe) [inline components/bpmn/bpmn-explorer-web/src/web/jaggery.conf]: Fixed in #471

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java (hunk 6 above)
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java [hunk 8]
```diff
@@ -441,6 +460,379 @@ public static String generateActivateLinkForProcessInfoPage(String processId) {
...
+        String type = null;
+        if (processDeployDetailsListType.getProcessEventsList() == null) {
+            type = "none";
+        } else if (processDeployDetailsListType.getProcessEventsList().getEnableEventsList() != null
+                && processDeployDetailsListType.getProcessEventsList().
+                getEnableEventsList().getEnableEvent() != null) {
+            List<String> events =
+                    Arrays.asList(processDeployDetailsListType.getProcessEventsList().
+                            getEnableEventsList().getEnableEvent());
+            if (events.contains("instanceLifecycle") && events.contains("activityLifecycle")
```
Comment (Subasinghe) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/BpelUIUtil.java]: Fixed in #471

**Pull request merged**